apps/pipeline-visualization/src/mermaid_utils.ts

```diff
@@ -41,36 +40,90 @@ function escapeHtml(unsafe: string) {
     .replace(/'/g, '&#039;');
 }
 
-export function createMermaidPipeline(
+export function processMermaidLinks(
```

Comment:
While the term `process` is correct, there is no real value in it. Try using more semantically concise variable names and especially method names.